### DISCUSSION TITLE
Where is the service that accepts requests

### DISCUSSION THREAD

**Discussion by hheart** [eclipse-theia/theia#14150] [Q&A]
hi.I would like to know where the request sent below will be accepted in the code,
![image](https://github.com/user-attachments/assets/cf67d72b-cb58-4f55-8a3a-dee480523a46)
it will send it in the browse like
![image](https://github.com/user-attachments/assets/2e9ce646-bbfd-4ff7-a4c0-7b8fe1ebdb2b)
thanks

**Comment by msujew** [comment]
Hey @hheart,

See here

https://github.com/eclipse-theia/theia/blob/19556f4d90c1b661ba53caea9b6a035a714e112d/packages/plugin-ext/src/main/browser/webview/webview-environment.ts#L74

And the code that calls the linked method.
